### PR TITLE
Link to infrastructure monitoring docs from Beats docs

### PR DESCRIPTION
**Pull request by dedemorton**
Added a link that points to the infrastructure monitoring docs.

Related to https://github.com/elastic/stack-docs/issues/224

### CODE REVIEW

**Comment by dedemorton** [comment]
Marked as "in progress" because the Filebeat and Metricbeat docs need updates, too.

File: libbeat/docs/dashboards.asciidoc [hunk 1]
```diff
@@ -9,6 +9,15 @@
 //// include::../../libbeat/docs/dashboards.asciidoc[]
 //////////////////////////////////////////////////////////////////////////
 
+ifdef::has_solutions[]
+TIP: The example dashboards are a great way to get started visualizing
```
Comment (bmorelli25) [inline libbeat/docs/dashboards.asciidoc]: This seems a bit long to me. I'm wondering if we can take out some of the words. Maybe even get rid of the first sentence?

> TIP: ~The example dashboards are a great way to get started visualizing {beatname_uc} data in {kib}.~ For deeper observability into your infrastructure, use the {infra-guide}/infrastructure-ui-overview.html[Infrastructure] and {infra-guide}/logs-ui-overview.html[Logs] UIs in {kib}. For setup details, see the {infra-guide}/index.html[Infrastructure Monitoring Guide].

**Comment by dedemorton** [comment]
@bmorelli25 Thanks! You're totally right. My first iteration is (hangs head in shame) often a bit wordy. :-P
